ci/jenkins/test.sh

```diff
@@ -1058,6 +1088,10 @@ if [[ ${TESTCASE} == "windows-install-ovs" ]]; then
 fi
 
```

Comment:
I mean you can put run_clean_kind here, otherwise clean_antrea could be executed twice for other tests.